scripts/generate_table.py

```diff
@@ -10,21 +10,21 @@
 with open(sys.argv[1], "r") as f:
     plugins = yaml.load(f, Loader=yaml.FullLoader)
 
-buffer = \
-    """
+buffer = """
 | | Name | Author | Description |
 |-|------|--------|-------------|"""
 
 plugin_template = """
-| ![Icon]({iconUrl}) | [{name}]({url}) | {author} | {description} |"""
+| ![Icon]({iconUrl}) | [{name}]({url}) | {authors} | {description} |"""
 
 for plugin in plugins:
-    plugin["description"] = plugin["description"] \
-        .replace("\n", " ") \
-        .replace("\r", "") \
-        .strip()
+    plugin["description"] = (
+        plugin["description"].replace("\n", " ").replace("\r", "").strip()
+    )
```

Comment:
Should we even display the description in the md table? That can be something for the individual md files, where we don't have to worry about wildly inconsistent row heights. Users will still have the short description.

Reply:
Good call, i just skimmed over this. We should use shortDescription instead.